packages/@lwc/synthetic-shadow/src/faux-shadow/events.ts

```diff
@@ -66,6 +67,10 @@ function targetGetter(this: Event): EventTarget | null {
     // Handle cases where the currentTarget is null (for async events),
     // and when an event has been added to Window
     if (!(originalCurrentTarget instanceof Node)) {
+        // If the event's target is being accessed async and originalTarget is not a keyed element, do not retarget
```

Comment:
we should add an issue to remove this in the future... with a TODO in the code.

Reply:
we should also do this in 222 too

Reply:
Filed #1511